fdbclient/SnapshotCache.h

```diff
@@ -33,15 +33,19 @@ struct ExtStringRef {
 
 	Standalone<StringRef> toStandaloneStringRef() {
 		auto s = makeString( size() );
-		memcpy( mutateString( s ), base.begin(), base.size() );
+		if (base.size() > 0) {
+			memcpy(mutateString(s), base.begin(), base.size());
+		}
 		memset( mutateString( s ) + base.size(), 0, extra_zero_bytes );
 		return s;
 	};
 
 	StringRef toArenaOrRef( Arena& a ) {
 		if (extra_zero_bytes) {
 			StringRef dest = StringRef( new(a) uint8_t[ size() ], size() );
-			memcpy( mutateString(dest), base.begin(), base.size() );
+			if (base.size()) {
```

Comment:
Minor nitpick but `base.size() > 0` would match the style of the other similar expressions.